### DISCUSSION TITLE
Buds+ : Tap and hold to mute not working

### DISCUSSION THREAD

**Discussion by rshaikh-lba** [timschneeb/GalaxyBudsClient#88] [Q&A]
When connected to my Samsung phone (and using the mandatory Wearable app), tap and hold will mute the earbuds. This is a great feature to having while most of us are working from home with kids and can't run back to the phone/computer to unmute yourself. It was the main reason for me getting the Buds+.

The tap and hold to mute doesn't happen on a PC. I am hoping  this amazing app can make that happen. I saw a discussion of the opposite problem where users didn't want to lift their hands from the keyboard to turn on the ambient sound.

I tried to assign a custom key press (F4 in my case) to the touchpad and attempt to mute that way, but it doesn't do anything. If I assign Ambient Sound, that does work. So I know the app is capturing the tap and hold event. In fact, if I can assign F4 to the tap and hold, that would blow the default behaviour out the window. This is because on the phone, when you tap and hold, you get a small beep that the mics are muted, but if you miss that, you won't know if you are on mute or not until someone says they can't hear you. By assigning F4, if the key has a light that let's you see if mute is enabled, we will be able to see it.

If assigning is not possible, I'll live with just muting the mics, better than nothing :) Thoughts on how easy it would be to implement?

Edit: I should add that if I assign F1 which is VolumeMute, it recognizes that event, as well as F2 and F3 for VolumeDown and VolumeUp. These work as intended. It's specifically MicMute (if that's the right term), that isn't being recognized.

**Comment by ntsour** [comment]
+1 This would be a GREAT addition. Very useful in pandemic/zoom world.
